jms-jpa/src/main/resources/application.properties

```diff
@@ -0,0 +1,54 @@
+## ---------------------------------------------------------------------------
+## Licensed to the Apache Software Foundation (ASF) under one or more
+## contributor license agreements.  See the NOTICE file distributed with
+## this work for additional information regarding copyright ownership.
+## The ASF licenses this file to You under the Apache License, Version 2.0
+## (the "License"); you may not use this file except in compliance with
+## the License.  You may obtain a copy of the License at
+##
+##      http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+## ---------------------------------------------------------------------------
+#
+# Quarkus
+#
+quarkus.banner.enabled = false
+quarkus.log.file.enable = true
+
+# Default Datasource
+quarkus.datasource.db-kind=h2
+quarkus.datasource.jdbc.max-size=8
+quarkus.datasource.jdbc.transactions=xa
+
+# Production Datasource
+#%prod.quarkus.datasource.db-kind=mysql
+#%prod.quarkus.datasource.username=admin
+#%prod.quarkus.datasource.password=admin
+#%prod.quarkus.datasource.jdbc.url=jdbc:mysql://localhost:3306/testdb
+#%prod.quarkus.datasource.jdbc.transactions=xa
+
+# Hibernate ORM
+quarkus.hibernate-orm.database.generation=drop-and-create
+#%prod.quarkus.hibernate-orm.database.generation=none
+
+# Quarkus Narayana JTA
+quarkus.transaction-manager.object-store-directory=target/narayana
+quarkus.transaction-manager.enable-recovery=true
+
+# Camel
+camel.rest.context-path=/api
+
+# Quarkus Artemis
+quarkus.artemis.enabled=true
+quarkus.artemis.devservices.enabled=false
```

Comment:
Why did you disable Artemis devservice? Datasource (H2) devservice is enabled and it make a confusion while I try to run `quarkus:dev`. I think that it will be better to keep it enabled as it is for Datasource service unless there is some technical problem.

Reply:
quarkus-artemis devservices depends on the docker enviromnent. But it is not availiable on Windows platform. So I have to disable it to make it passing tests on the windows. Alternatively I think it should be better to change to `%test.quarkus.artemis.devservices.enabled=false`, then `quarkus:dev` should work. I will update it.